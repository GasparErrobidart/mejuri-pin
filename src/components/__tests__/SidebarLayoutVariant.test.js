import '@testing-library/jest-dom'
import 'jest-styled-components'

import React         from 'react'
import { cleanup }   from '@testing-library/react'
import renderer      from 'react-test-renderer'

import SidebarlayoutVariant   from '../SidebarlayoutVariant'


afterEach(cleanup)

test('Renders "SidebarlayoutVariant" component', () => {
  const component       = renderer.create(
    <SidebarlayoutVariant>
      <div>
        sidebar
      </div>
      <div>
        content
      </div>
    </SidebarlayoutVariant>
  );
  const tree            = component.toJSON()
  expect(tree).toMatchSnapshot();
});
