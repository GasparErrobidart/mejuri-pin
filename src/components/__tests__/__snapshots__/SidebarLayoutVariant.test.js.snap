// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders "SidebarlayoutVariant" component 1`] = `
.c0 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-align-items: stretch;
  -webkit-box-align: stretch;
  -ms-flex-align: stretch;
  align-items: stretch;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
}

@media (min-width:768px) {
  .c0 {
    -webkit-flex-direction: row;
    -ms-flex-direction: row;
    flex-direction: row;
  }

  .c0 > *:first-child {
    padding-top: 30px;
    -webkit-flex: 0 0 250px;
    -ms-flex: 0 0 250px;
    flex: 0 0 250px;
  }

  .c0 > *:last-child {
    -webkit-box-flex: 1;
    -webkit-flex-grow: 1;
    -ms-flex-positive: 1;
    flex-grow: 1;
  }
}

<div
  className="c0"
>
  <div>
    sidebar
  </div>
  <div>
    content
  </div>
</div>
`;
